src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseDataService.java

```diff
@@ -303,20 +303,39 @@ public void updateCaseData(CaseData caseData, UUID stageUUID, Map<String, String
         log.info("Updated Case Data for Case: {} Stage: {}", caseData.getUuid(), stageUUID, value(EVENT, CASE_UPDATED));
     }
 
-    void updateDateReceived(UUID caseUUID, UUID stageUUID, LocalDate dateReceived, int days) {
 
+    void updateDateReceived_defaultSla(UUID caseUUID, UUID stageUUID, LocalDate dateReceived) {
```

Comment:
I like that these are seperated, out of scope for this maybe but would be nice in there own class...